frontend/src/styles/PlayerAnswerButton.module.scss

```diff
@@ -0,0 +1,144 @@
+.answerButtonBlueStar {
+    position: absolute;
+    width: 500px;
+    height: 300px;
+    left: 745px;
+    top: 125px;
```

Comment:
try to use `rem` as much as possible to create a responsive UI on different screen size. Exception would be border radius, border width, or elements you want to have a fixed size across different screen sizes.